bigframes/streaming/dataframe.py

```diff
@@ -291,13 +291,13 @@ def __repr__(self, *args, **kwargs):
 
     __repr__.__doc__ = _curate_df_doc(inspect.getdoc(dataframe.DataFrame.__repr__))
 
-    def _repr_html_(self, *args, **kwargs):
-        return _return_type_wrapper(self._df._repr_html_, StreamingDataFrame)(
+    def _repr_html_fallback_(self, *args, **kwargs):
```

Comment:
We don't need the trailing `_` because we aren't trying to get ipython to automatically call this. In fact, we do not want it to call this directly.